### PR TITLE
group_id for SimChannel

### PR DESCRIPTION
**Pull request by twybo**
**Issue being addressed**
Added the option to pass a group id to a SimChannel, which was not yet implemented.

Will quickly add a commit for changelog

### CODE REVIEW

File: NuRadioReco/framework/sim_channel.py [hunk 1]
```diff
@@ -14,11 +14,11 @@ class SimChannel(NuRadioReco.framework.channel.Channel):
     """
     Object to store simulated channels.
 
-    This class is the same as the regular channel trace but has apart from the channel id also
+    This class is the same as the regular channel trace but has apart from the channel id (and possible group id) also
     a shower and ray tracing solution id
     """
 
-    def __init__(self, channel_id, shower_id, ray_tracing_id):
+    def __init__(self, channel_id, shower_id, ray_tracing_id, channel_group_id):
```
Comment (sjoerd-bouma) [inline NuRadioReco/framework/sim_channel.py:21]: Please set the default argument to None (just copy what is done in Channel on the LOFAR branch). An remember to update the docstring accordingly (you can also copy that from the LOFAR branch docstring if you like)

File: changelog.txt [hunk 2]
```diff
@@ -11,6 +11,7 @@ It comes with a class to handle responses of the different signal chain componen
 added new STATUS logging level by default, added logging documentation and moved logging
 module to NuRadioReco/utilities folder. Logging functionality from NuRadioReco/modules/base
 is now deprecated.
+- SimChannel can now take a group id
```
Comment (sjoerd-bouma) [inline changelog.txt:14]: It probably makes more sense to update the changelog once, on the LOFAR branch, so I think this can be removed for now : )

**Review by sjoerd-bouma — COMMENTED** [review]
Thanks! Can you please change the target branch to `LOFAR` (instead of `develop`)? This feature won't work before it's implemented for the `Channel` too, which is done on that branch.

File: NuRadioReco/framework/sim_channel.py (hunk 1 above)
Comment (twybo) [reply NuRadioReco/framework/sim_channel.py]: I set the default argument and copied the docstring from the LOFAR branch, thanks.

**Comment by MijnheerD** [comment]
So there are really only 5 commits here that differ from the LOFAR branch... Not sure why the PR is not updating, on my local version git shows this correctly. I think this can be merged into LOFAR without any issues, or should start a new PR just to be safe?

**Comment by MijnheerD** [comment]
I think what I am going to do is to merge this locally into LOFAR, push the changes, and close this PR. I don't understand why Github says there are 53 commits to be merged into LOFAR, this should not be the case. Is there anything against this procedure?

**Comment by MijnheerD** [comment]
Ah damn, I forgot to merge this into LOFAR before merging that branch in develop. Is it okay to merge this into develop now @sjoerd-bouma ?

**Review by cg-laser — APPROVED** [review]

**Pull request merged**